.github/workflows/ci.yml

```
@@ -664,113 +664,118 @@ jobs:
         env:
           GITHUB_WORKSPACE: ${{ github.workspace }}
           GITHUB_REPOSITORY: ${{ github.repository }}
+          NEEDS_FULL_SETUP: ${{ matrix.needs_full_setup }}
         run: |
-          # Add .env file and create artifacts directory
-          # Note: the wonky indentation is intentional so there is no space at the start of
-          # each newline in the .env file
-          if [[ "${{ matrix.db }}" =~ mysql ]]; then
-            if [[ "${{ matrix.db }}" == "mysql57pdo" ]]; then
+          # Artifacts directory must be created after composer install as it would remove the artifacts directory
+          # This seems a bit strange, if you need to add an artifact at an earlier stage then revalidate that
+          # composer install does actually remove the artifact directory
+          mkdir artifacts
+
+          if [[ NEEDS_FULL_SETUP == "false" ]]; then
```

```suggestion
          if [[ $NEEDS_FULL_SETUP == 'false' ]]; then
```
Need a `$` for the variable :-)

Using single quotes just to copy style above where a conditional check was done `if [[ $SUCCESSFULLY_REQUIRED == 'false' ]]; then`

---

The single/double quotes isn't consistent everywhere in this file (I copied double quotes from somewhere in here too) but I have no issues with that change.

Done.